qiita_pet/webserver.py

```diff
@@ -106,6 +104,7 @@ def __init__(self):
             (r"/study/sharing/", ShareStudyAJAX),
             # ORDER FOR /study/description/ SUBPAGES HERE MATTERS.
             # Same reasoning as below. /study/description/(.*) should be last.
+            (r"/study/description/sample_template/", SampleTemplateAJAX),
```

Comment:
Shouldn't the endpoints and collections described in the docstrings be expressed here?